tools/i18n/chart_config_extractor.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script to generate a .pot file from the chart config, used for translations.
+
+To run: python3 tools/i18n/chart_config_extractor.py
+
+This will produce /server/l10n/chart_titles.pot
+Then, run pybabel init -l $LOCALE -i server/l10n/chart_titles.pot -d server/l10n -D chart_titles
+which will initialize a chart_titles.po file for the locale.
```

Comment:
This is just for development right? We'll eventually make .po files from TC output?

Reply:
yep